Reference/Management/Services/ContentService/Index-v7.md

```diff
@@ -0,0 +1,46 @@
+---
+keywords: services content service
+versionFrom: 7.0.0
+versionRemoved: 8.0.0
```

Comment:
@dampee - By adding the `versionRemoved`, isn't it going to say that the ContentService has been removed in Umbraco 8? It hasn't been removed, has it? 🤔

Reply:
@sofietoft Let me verify that. :-o

Reply:
It will mention 7.0.0 - 8.0.0 instead of 7.0.0+
So I think the change is ok.